tests/testserver/server.py

```diff
@@ -25,10 +25,11 @@ class Server(threading.Thread):
     """Dummy server using for unit testing"""
     WAIT_EVENT_TIMEOUT = 5
 
-    def __init__(self, handler, host='localhost', port=0, requests_to_handle=1, wait_to_close_event=None):
+    def __init__(self, handler=None, host='localhost', port=0,
```

Comment:
Added an argument briefly (which made this rather long, though now I remember that one line in the contributor's guide), then removed it, and forgot to revert the formatting change. Apologies. Will revert.